src/main/resources/upgrade-files/4.1.1_5.0/rice_server/rice-server-script.xml

```diff
@@ -21,6 +21,9 @@
 	<changeSet author="RICE20" id="RICE_SERVER_UPDATES_oracle" dbms="oracle">
```

Comment:
Thank you for catching this.  Rolled back changes to unused script 'src/main/resources/upgrade-files/4.1.1_5.0/rice_server/rice-server-script.xml'